version.hh

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#define _showVersion(program,g_gitHash) { \
```

Comment:
Please make this an inline function, will work just as well.

Reply:
Sure enough.